src/app/(main-layout)/resources/[id]/page.tsx

```diff
@@ -1,12 +1,17 @@
-import { getTranslations } from "next-intl/server";
 import Api from "@/lib/api";
 import PageHeader from "@/components/PageHeader";
 import ResourceCard from "../_components/ResourceCard";
-import { getSession } from "@/lib/session";
+import { getSession, getUser } from "@/lib/session";
+import Divider from "@/components/Divider";
+import ResourceComments from "./_components/ResourceComments";
+import UnauthorizedCommentArea from "./_components/UnauthorizedCommentArea";
+import CommentArea from "./_components/CommentArea";
+import { Suspense } from "react";
+import ResourcesLoadingPlaceholder from "../_components/ResourcesLoadingPlaceholder";
 
 export default async function Page({ params }: { params: { id: string } }) {
-  const t = await getTranslations();
   const session = await getSession();
+  const user = await getUser();
```

Comment:
Since you already have the session this could be:
```suggestion
  const user = session?.user;
```